modules/elt/pages/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage.adoc

```diff
@@ -0,0 +1,212 @@
+= Use Airbyte to load data from external sources to Teradata Vantage
+:experimental:
+:page-author: Krutik Pathak
+:page-email: krutik.pathak@teradata.com
+:page-revdate: June 9th, 2023
+:description: Use Airbyte with Teradata Vantage.
+:keywords: airbyte, data warehouses, compute storage separation, teradata, vantage, cloud data platform, object storage, data transfer, data extraction, enterprise analytics, elt.
+:tabs:
+:dir: getting-started-with-airbyte
+
+== Overview
+
+This tutorial showcases how to use Airbyte (an open-source Extract Load Transform tool) with Teradata Vantage. We work with a very simple end-to-end setup to load data from Google Sheets to Teradata Vantage using Airbyte.
+
+
+image::{dir}/sample_employees_payrate_google_sheets.png[Sample Employees Payrate Google Sheets, width=75%]
+
+* Source: Google Sheets
+* Destination: Teradata Vantage
+
+== Prerequisites
+* Access to a Teradata Vantage Instance. This will be defined as the destination in Airbyte. You will need a database `Host`, `Username`, and `Password` for Airbyte’s configuration.
```

Comment:
```suggestion
* Access to a Teradata Vantage Instance. This will be defined as the destination of the Airbyte connection. You will need a database `Host`, `Username`, and `Password` for Airbyte’s configuration.
```